service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java

```diff
@@ -102,12 +102,11 @@ public static RegistryManager createFromConnectionString(String connectionString
      * @deprecated because this method declares a thrown IOException even though it never throws an IOException. Users
      * are recommended to use {@link #RegistryManager(String, RegistryManagerOptions)} instead since it does not declare this exception even
      * though it constructs the same RegistryManager.
-     * @throws IOException This exception is never thrown.
      */
     @Deprecated
     public static RegistryManager createFromConnectionString(
             String connectionString,
-            RegistryManagerOptions options) throws IOException
```

Comment:
Very publicly used method, so this would be a breaking change for a lot of users

Reply:
Same for the public methods in this class